faq.md

```diff
@@ -0,0 +1,113 @@
+Frequently Asked Questions
+
+## What is TED AI Open Source Hackathon?
+The [TED AI Open Source Hackathon](https://ai-event.ted.com/hackathon) is an exciting programming event that challenges you to find ways to use Generative AI to tackle social and environmental issues. You will have the opportunity to use your coding skills and collaborate with fellow developers, all in the name of good.
+
+## When and where will the hackathon take place?
+The hackathon is scheduled for Oct 19 and 20, 2024 at Microsoft Reactor in San Francisco, CA. This is the weekend before the TED AI conference.
+
+### Site Address: 
+Microsoft Reactor 
+555 California Street, Suite 200 
+San Francisco, CA 94104 
+
+Reception is on the 2nd floor, hackathon will be on the 3rd floor. 
+
+### Parking: 
+
+ - 555 California Garage  
+   Entrance Address: 424 Pine St  
+   Hourly Rate: $16  
+   Daily Rate: $35  
+ - 550 Kearny Garage  
+   Entrance Address: 655 Sacramento St  
+   Hourly Rate: $15  
+   Daily Rate: $30  
+- Public Transportation  
+   BART/Muni: Montgomery Station – 0.3 miles, 5 minute walk  
+   Caltrain – San Francisco Station – 1.5 miles, 20 minute drive  
+
+### Nearby Hotels:
+
+- Omni San Francisco Hotel  
+   500 California St, San Francisco, CA 94104  
+   4 star hotel  
+   0.1 mile – 3 minute walk  
+- Grant Plaza Hotel  
+   465 Grant Ave, San Francisco, CA 94108  
+   2 star hotel  
+   0.1 mile – 3 minute walk  
+- Four Seasons Hotel San Francisco at Embarcadero  
+   222 Sansome St, San Francisco, CA 94104  
+   5 star hotel  
+   0.2 mile – 4 minute walk  
+- Stanford Hotel  
+   905 California St, San Francisco, CA 94108  
+   4 star hotel  
+   0.4 mile – 8 minute walk  
+
+
+## What is the hackathon timeline?
+
+TBD: Application open  
+Sep 6: Application close  
+Sep 20: Teams announced  
+Oct 19: Hackathon start  
+Oct 20:  Hackathon end  
+
+## How can I participate in the hackathon?
+Register your team at https://ai-event.ted.com/hackathon.  We will then evaluate their submission and let them know. The space is limited so don't wait.
```

Comment:
Hackathon page is https://tedai-sanfrancisco.ted.com/hackathon/

Reply:
75a5b311cac6e25ccad6dade58ace598e559553d